setup.sh

```diff
@@ -1,36 +1,36 @@
 #!/usr/bin/env bash
 set -e
 
-HERE=${BASH_SOURCE%/*}
-source ${HERE}/vault_auth.sh
-
-echo "Installing Pip and required Python packages"
-apt-get install python3-pip -y > /dev/null
-pip3 install -q -r ${HERE}/requirements.txt
-
-echo "-------------------------------------------"
-echo "Installing bb8 user and group"
-
-if ! id -u bb8 > /dev/null 2>&1; then
-    useradd bb8 -U -d /var/lib/bb8
-    password=$(vault read --field password secret/backup/bb8/user)
-    echo "bb8:$password" | chpasswd
-
-    mkdir -p /var/lib/bb8/.ssh
-    
-    # add to docker group
-    usermod -aG docker bb8
+if [[ ($# -eq 0) ]]; then
+    echo "Usage: ./setup.sh PATH_TO_SOURCE_CONFIG [TARGET ...]"
+    exit -1;
 fi
 
-# give bb8 group ownership of this dir
-chgrp -R bb8 $HERE
-chmod -R g+w $HERE
-
-echo "-------------------------------------------"
-echo "Running setup.py:"
-(cd "$HERE" && su -c ./setup.py bb8)
+source_config_path=$1
+shift && targets="$@"   # Targets is all the args after the first
 
-echo "-------------------------------------------"
+# Get secrets and put them in secrets/
+# (where the Dockerfile looks for them)
+HERE=${BASH_SOURCE%/*}
+source ${HERE}/vault_auth.sh
 
-echo "Setup complete. To schedule backups, run sudo ./schedule.py"
-echo "To perform a restore, run ./restore.py"
\ No newline at end of file
+function delete_secrets {
+    set +e
+    rm -r ${HERE}/secrets
+}
+trap delete_secrets SIGINT EXIT
+./obtain_secrets.py
```

Comment:
should we use the `$HERE` in this path and also in the call to `./bb8` below, so that this script can be run from anywhere?